docs/installation.rst

```diff
@@ -1,15 +1,17 @@
 Installation
 ============
 
+.. contents::
+   :local:
+
 Requirements
 ************
 
-- In order to build images, you need to have either ``podman`` or ``docker``
-  installed as well as the ``ansible-builder`` Python package.
-  The ``--container-runtime`` option needs to correspond to the Podman/Docker
-  executable you intend to use.
-- ``ansible-builder`` version ``1.1.0`` requires Python ``3.8`` or higher.
-
+- To build images, you must install either ``podman`` or ``docker``
+  as well as the ``ansible-builder`` Python package.
+- The ``--container-runtime`` option needs to correspond to the Podman/Docker
+  executable you intend to use (where is this set?).
+- ``ansible-builder`` version ``3.x`` requires Python ``3.8`` or higher.
```

Comment:
`3.9` is the minimum. See requirements in `setup.cfg`.

Reply:
Thanks for the review @Shrews! I think I have addressed your concerns and your answers in 4b5a9fd85759ecd7dcb4c4aad197de99a6e3b6f1 and 09bb9aff1880a7bc17822599191901d2ed584c4d.